doc/Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS   =
-SPHINXBUILD  = python ../sphinx-build.py
+SPHINXBUILD  = python ../sphinx/cmd/build.py
```

Comment:
@stephenfin @tk0miya this change means that a Sphinx as currently installed system wise is used to build the docs. This causes me problems for example for latexpdf, where I use building the docs also as a test when developing features. As far as I know, basically I now have to always work in a virtualenv. This means whenever I make a small change I have to `pip install --upgrade . ` and this takes time. In fact I don't do that and recreated the scripts but here I have to also modify the Makefile which is under git control.

Why not still having the scripts (build and quickstart at least) in the repo but simply avoid including them in packaging for PIP?

Reply:
For example at current master v1.7+ f07b08033

I get when trying to build the html docs:
```
/PATH/sphinx/doc/domains.rst:849: WARNING: Unknown interpreted text role "cpp:expr".
```
because I have Sphinx 1.6.3 in my system-wise environment. It is a bit annoying to have to re-install Sphinx master in a virtual env only to test its docs build fine.